easybuild/easyconfigs/q/qsimcirq/qsimcirq-0.14.0-foss-2022a.eb

```diff
@@ -0,0 +1,44 @@
+##
+# Author:    Wahid Mainassara <wahid.mainassara@lxp.lu>
+##
+easyblock = 'PythonBundle'
+
+name = 'qsimcirq'
+version = '0.14.0'
+
+homepage = 'https://quantumai.google/qsim'
+description = """qsim and qsimh are Schrödinger and Schrödinger-Feynman
+ simulators for quantum circuits."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('cirq-core', '1.0.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('pip', '22.2.2', {
+        'use_pip': False,
+        'checksums': ['3fd1929db052f056d7a998439176d3333fa1b3f6c1ad881de1885c0717608a4b'],
+    }),
+    ('wheel', '0.37.1', {
+        'checksums': ['e9a504e793efbca1b8e0e9cb979a249cf4a0a7b5b8c9e8b65a5e39d49529c1c4'],
+    }),
+    ('setuptools', '65.3.0', {
+        'checksums': ['7732871f4f7fa58fb6bdcaeadb0161b2bd046c85905dbaa066bdcbcc81953b57'],
+    }),
+    ('absl-py', '1.2.0', {
+        'modulename': 'absl',
+        'checksums': ['f568809938c49abbda89826223c992b630afd23c638160ad7840cfe347710d97'],
+    }),
+    (name, version, {
+        'source_tmpl': '%(name)s-%(version)s-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl',
```

Comment:
@Wahid612 A (big) downside of using a pre-built binary wheel is that it will only work on `x86_64` (not on `arm64`, etc.).

Did you try a build from source using the [sources available from GitHub](https://github.com/quantumlib/qsim)?